window-manipulate/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "applications": {
```

Comment:
In https://github.com/mdn/webextensions-examples/issues/71 we decided to not specify IDs for new extensions. This will help preserve compatibility with Chrome. You can remove this entire `applications` key from the manifest.